IndianList/cdn_block.txt

```diff
@@ -67,28 +67,23 @@
 ||fdcdn.prixacdn.net/media/1200x_150px.gif
 ||fdcdn.prixacdn.net/media/global_hr_solution_desktop.gif
 ||fdcdn.prixacdn.net/media/NHPF.gif
-||goo.gl/iqmWie
+||goo.gl/iqmWie^
 ||googleapis.com/production-prothomalo/contents/uploads/media/2020/03/21/e228ecefda1d4ffaf696fdbf60b12bd4-5e75b7d59efd0.gif
 ||googleapis.com/production-prothomalo/contents/uploads/media/2020/03/22/a53c582bdeb69a2421656d4c824cec7b-5e77374fd1e5a.png
 ||hostitbd.com/wp-content/uploads/2020/02/website-create-by-hostitbd.png
-||i.ibb.co/64NYFCq/Whats-App-Image-2020-03-06-at-2-34-08-PM.jpg
-||i.ibb.co/dPqxFdk/Whats-App-Image-2020-02-05-at-3-45-48-PM.jpg
-||i.ibb.co/HnmRMHv/Whats-App-Image-2020-02-02-at-5-39-22-PM.jpg
-||i.ibb.co/nkk4q0C/Whats-App-Image-2020-02-25-at-5-21-09-PM.jpg
-||i.ibb.co/Wt2cW3r/Whats-App-Image-2020-02-25-at-5-20-30-PM.jpg
-||i.imgur.com/nu4zSbR.jpg
+||i.ibb.co/*/Whats-App-Image-2020-*-at-*.jpg
```

Comment:
these can't be simply compressed as many content image files are with the same pattern.